src/panels/templates/AutomaticCreateCluster.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
💡 observation: I think this is stock standard json might be get an eye for anyone who might have anything specific to say. or fyi @qpetraroia